docs/tutorial/fastapi.rst

```diff
@@ -0,0 +1,419 @@
+.. _build-a-rock-for-a-fastapi-application:
```

Comment:
Should not there be a link to this tutorial in `docs/tutorial/index.rst`?

Reply:
yes will add

Reply:
should be fixed